js/src/main/scala/tutorial/App.scala

```diff
@@ -40,13 +40,13 @@ object App extends js.JSApp {
       )
 
     /* render root directory */
-    browser.fetchPathsUnder(Root)
+    browser.fetchPathsUnder(RootRef)
   }
 }
 
-final case class State(wantedPath: Path, result: Try[LookupResult])
+final case class State(wantedPath: PathRef, result: Try[Either[LookupError, Seq[PathRef]]])
```

Comment:
jeg tenkte en av de første oppgavene kunne være å implementere støtte for å vise frem filer, og da syntes jeg at `LookupResult` leder i feil retning, gitt at det blir en annen suksess-type